build/yamls/antrea-aks.yml

```diff
@@ -1088,6 +1088,8 @@ data:
     # Default MTU to use for the host gateway interface and the network interface of each Pod.
     # If omitted, antrea-agent will discover the MTU of the Node's primary interface and
     # also adjust MTU to accommodate for tunnel encapsulation overhead (if applicable).
+    # 1450 is the default value for IPv4. With IPv6 setup, its value is usually 20 less than IPv4 one
```

Comment:
Since we do MTU auto discovery, I think no need to explain the 1450 default value.

Why IPv6 MTU must be larger than (BTW I guess you mean "not be smaller than") 1280?

Reply:
Yes, no smaller than 1280. 1280 is the common minimum value for IPv6 MTU. Do you think there's no need to mention it? But at least I think a 20 bytes difference between IPv4 and IPv6 MTU should be mentioned.

Reply:
I feel the problem is if MTU is auto-detected, and 1450 is just one possible value when VXLAN or Geneve is used, and we do not mention other possible values, why we specifically mention 20 bytes for IPv6 with encap/hybrid modes only.
If you want to describe this, I guess you need more words to make it very clear.

Reply:
Now I got it. Thank you! Let's remove this line of comment.

Reply:
Please take another look.